pkg/sql/opt/optgen/cmd/optfmt/main.go

```diff
@@ -402,16 +402,16 @@ func (p *pp) docOnlyExpr(e lang.Expr) pretty.Doc {
 		))
 
 		inner := pretty.Group(pretty.Fold(pretty.Concat,
-			binding,
+			pretty.NestT(binding),
 			pretty.Line,
-			p.docExpr(e.Result),
+			pretty.NestT(p.docExpr(e.Result)),
```

Comment:
There may be a bug in `pretty` that prevents this `pretty.NestT` from indenting correctly. Trying to figure out a work-around.